package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "auslese",
-    "version": "2.4.1",
+    "version": "2.5.0",
```

The version within the `package.json` will be automatically updated during the release process via the tool `np` (https://github.com/sindresorhus/np)

```suggestion
    "version": "2.4.1",
```